.gitignore

```diff
@@ -1,29 +1,30 @@
-**/venv/
-**/.idea/
-
-src/gui/node_modules/
-src/gui/dist/
-src/core/__pycache__/
-src/core/migrations/versions/__pycache__/
+# built and cached files
+venv/
+__pycache__/
+*.pyc
+node_modules/
+dist/
+build/
 
+# temporary files of editors
 .DS_Store
-.*.swp
 .~
+*.*~
+*.bak
+*.swp
 
-node_modules
-/dist
-
+# sensitive data not to be commited
 .env.local
 .env.*.local
+*.pem
+*.key
+*.log
 
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-
-.idea
-.vscode
+# settings of editors
+.idea/
+.vscode/
 *.suo
 *.ntvs*
 *.njsproj
 *.sln
-*.sw?
```

Comment:
see the comment above

Reply:
I didn't change that line - the newline was missing here